vendor/migrations/0019_fill_uuid_address_profile.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.1.3 on 2021-01-07 23:10
+
+from django.db import migrations
+
+import uuid
+
+def generate_address_uuid(apps, schema_editor):
+    AddressModel = apps.get_model('vendor', 'Address')
```

Comment:
there are probably way too few entries in the db to actually worry about this - but should we be generated uuids checking against one another before setting them?

Reply:
No matter this the proper way for adding uuids fields to models. I don't think anyone else has used django-vendor put if they did they need the three migrations for it to work.
1. migration added the fields with blank and null to true.
2. migrations to generate the uuids for fields.
3. migration to lock the uuid field with null and blank to False